CHANGES.md

```diff
@@ -8,6 +8,9 @@
 
 ### Fixed and improved
 
+* Root Marathon support post-installation configuration of flags and JVM settings has been improved. (DCOS_OSS-3556)
```

Comment:
I am not a native speaker but the word 'support' is kind of extra for me in that sentence, I would drop that...

Reply:
missing a `for` :) thanks.